### PR TITLE
DEV-3689: Add header logo alt text field to DonationPage model

### PR DESCRIPTION
**Pull request by GuiMend**
Please fill out each section even if it's just with "N/A"

#### Plan? (if this draft/incomplete indicate what you intend to do and how)
n/a

#### What's this PR do?
Adds `header_logo_alt_text` field to DonationPage model
#### Why are we doing this? How does it help us?
This adds the ability for users to set alt text on their header logo.
#### How should this be manually tested? Please include detailed step-by-step instructions.
- Check Donation Page in admin, `header_logo_alt_text` field should be editable
#### Did this PR make changes to the user interface that may require changes to the user-facing docs?
no
#### Have automated unit tests been added? If not, why?
yes
#### How should this change be communicated to end users?
n/a

#### Are there any smells or added technical debt to note?
no
#### Has this been documented? If so, where?
no
#### What are the relevant tickets? Add a link to any relevant ones.
https://news-revenue-hub.atlassian.net/browse/DEV-3689
#### Do any changes need to be made before deployment to production (adding environment variables, for example)?
no
#### Are there next steps? If so, what? Have tickets been opened for them? List next-step tickets here:
Parent ticket: https://news-revenue-hub.atlassian.net/browse/DEV-3056

### CODE REVIEW

File: apps/pages/migrations/0009_donationpage_header_logo_alt_text.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.19 on 2023-07-07 17:59
```
Comment (njharman) [inline apps/pages/migrations/0009_donationpage_header_logo_alt_text.py:1]: Our standard is that migration filenames include the DEV-3689 ticket number. see other migration files.

**Review by njharman — CHANGES_REQUESTED** [review]
One file name change. Pre-review checklist is not complete.

AC3 There is not explicit test that PATCHes this field. Although, I don't think we have any tests like that.

File: apps/pages/migrations/0009_donationpage_header_logo_alt_text.py (hunk 1 above)
Comment (GuiMend) [reply apps/pages/migrations/0009_donationpage_header_logo_alt_text.py]: Done!

**Review by njharman — APPROVED** [review]
looks good

**Pull request merged**